python/tvm/tir/expr.py

```diff
@@ -577,6 +577,8 @@ def __ne__(self, other):
             return self.value != other.value
         return self.value != other
 
+    __hash__ = PrimExpr.__hash__
```

Comment:
Better to be a method like `__eq__` and `__ne__`?
```suggestion
    def __hash__(self):
        return PrimExpr.__hash__(self)
```

Reply:
I agree :)